core/modules/catalog/helpers/index.ts

```diff
@@ -112,6 +112,13 @@ export function attributeImages (product) {
       }
     }
   }
+  if (attributeImages.length <= 0) {
+    attributeImages.push({
+      'src': getThumbnailPath(product['image'], config.products.gallery.width, config.products.gallery.height),
```

Comment:
Let's replace `'src'` with just `src` and `product['image']` with just `product.image`, same for these 2 below

Reply:
This PR is also related to your other one: https://github.com/DivanteLtd/vue-storefront/pull/5091
What do you think about merging them in one and getting rid of hardcoded sizes in every place inside `attributeImages`? (not only in if (product[attribute]))

Reply:
Sure, will do the changes.

Reply:
@Fifciu I have updated the code.